--- a/src/Element/SingleDateTime.php
+++ b/src/Element/SingleDateTime.php
@@ -134,7 +134,7 @@ class SingleDateTime extends FormElement {
     $element['#attributes']['autocomplete'] = 'off';
 
     // Prevent keyboard on mobile devices.
-    $element['#attributes']['onfocus'] = 'blur();';
+    // $element['#attributes']['onfocus'] = 'blur();';
 
     // Attach library.
     $element['#attached']['library'][] = 'single_datetime/datetimepicker';
