diff --git a/unique_field_ajax.module b/unique_field_ajax.module
index 1184c42..306d453 100644
--- a/unique_field_ajax.module
+++ b/unique_field_ajax.module
@@ -149,8 +149,9 @@ function _unique_field_ajax_entity_form_builder(
     'per_lang',
     'use_ajax',
     'case_sensitive',
     'no_enforce',
     'message',
+    'message_warning',
   ];
   foreach ($fields as $field) {
     $value = $form_state->getValue(['unique_field_ajax', $field]);
