diff --git a/unique_field_ajax.module b/unique_field_ajax.module
index 1184c42..306d453 100644
--- a/unique_field_ajax.module
+++ b/unique_field_ajax.module
@@ -132,6 +132,7 @@ function _unique_field_ajax_entity_form_builder(string $entity_type, EntityInter
     'case_sensitive',
     'no_enforce',
     'message',
+    'message_warning',
   ];
   foreach ($fields as $field) {
     $value = $form_state->getValue(['unique_field_ajax', $field]);
