Index: web/modules/contrib/empty_fields/empty_fields.module
<+>UTF-8
===================================================================
--- web/modules/contrib/empty_fields/empty_fields.module
+++ web/modules/contrib/empty_fields/empty_fields.module
@@ -65,7 +65,7 @@
     if ($entity->get($field_name)->isEmpty() && ($handler = $display->getRenderer($field_name)->getThirdPartySetting('empty_fields', 'handler'))) {
       // @todo Get render with react() method.
       if ($handler === 'nbsp') {
-        $markup = ['#markup' => '&nbsp;'];
+        $markup = ['#markup' => '–'];
       }
       /** @var \Drupal\Core\Field\FieldDefinitionInterface $definition */
       $definition = $entity->get($field_name)->getFieldDefinition();
