diff --git a/js/garland.js b/js/garland.js
--- js/garland.js
+++ js/garland.js
@@ -14,7 +14,9 @@
 
       // if garland not exists.
       if ($('#garland').length === 0) {
-        $('body').prepend('<div id="garland"></div>');
+        const newDiv = document.createElement("div");
+        newDiv.id = "garland";
+        $('header.content-header').prepend(newDiv);
       }
 
       if ($('#garland').length > 0) {
@@ -24,6 +26,7 @@
         } else {
           // eslint-disable-next-line
           $('#garland').css('position', 'absolute');
+          $('#garland').css('margin-top', '-2rem');
         }
 
         // if core toolbar or admin_toolbar exists.
